samples/transferBtc-samples.js

```diff
@@ -314,4 +324,5 @@ const printP2PKHTransferHashWithNotEnoughForFees = async () => {
     await printP2PKHTransferHashWith0Change();
     await printP2PKHTransferHashWithChange();
     await printP2PKHTransferHashWithNotEnoughForFees();
+    await printBlockHeader()
```

Comment:
I don't think we need to add this sample here or to add this sample anywhere at all. Specially on this file which only shows samples of `transferBtc`. I think you can remove the changes from this file.